Client/functions.js

```diff
@@ -3,5 +3,22 @@ import { MAX_NAME_SIZE } from "./constants.js";
 // Validates an input name
 // Currently, only specifications is that the type is a string and name is not empty but also not too long.
 export function isValidName(name) {
-    return typeof (name) === "string" && name.length > 0 && name.length <= MAX_NAME_SIZE;
+    return typeof name === "string" && name.length > 0 && name.length <= MAX_NAME_SIZE;
 }
+
+// Returns array of all tiles that are within a given radius of a coordinate
+// Coordinate should have integer values, and radius should be nonnegative
+// minX, minY, maxX, maxY are optional parameters that can be given to filter results
+export function withinRadius(c, r, minX, minY, maxX, maxY){
```

Comment:
Instead of checking for `undefined`, you could give the min and max parameters default values:
```suggestion
export function withinRadius(c, r, minX = -Infinity, minY = -Infinity, maxX = Infinity, maxY = Infinity){
```
You may even want to put this extra functionality into a separate function like `inBounds` because it's not something you would expect from a function named `withinRadius`.

Reply:
I'll keep the functionality just in case we have some crazy large radii in the future.

(Implemented default values and renamed)